### PR TITLE
#98 Add `dotenv` format option to `config get` command

### PR DESCRIPTION
**Pull request by imadphp**
Issue: #98 

Example:

```
╰─ wp config list --format=dotenv
WP_OLD_HASH_PREFIX="$P$"
WP_ENV="development"
WP_HOME="http://example.dev"
WP_SITEURL="http://example.dev/wp"
CONTENT_DIR="/app"
(...)
WP_DEBUG_DISPLAY="1"
SCRIPT_DEBUG="1"
SAVEQUERIES="1"
WP_DEBUG="1"
```

### CODE REVIEW

File: src/Config_Command.php [hunk 1]
```diff
@@ -856,4 +864,20 @@ private function parse_separator( $separator ) {
...
 	}
+
+	/**
+	 * Writes a provided variable's key and value to stdout, in dotenv format.
+	 *
+	 * @param array $value
+	 */
+	private function print_dotenv ( array $value )
+	{
+		if ( ! isset( $value['name'] ) || ! isset( $value['type'] ) || 'constant' !== $value['type'] ) {
```
Comment (imadphp) [inline src/Config_Command.php]: @schlessera Do you agree that only constant should be supported here?

**Comment by imadphp** [comment]
Any plans to merge this sometime?

**Comment by imadphp** [comment]
@schlessera

File: src/Config_Command.php (hunk 1 above)
Comment (schlessera) [reply src/Config_Command.php:874]: Yes, I agree.

**Comment by schlessera** [comment]
Thanks for the PR, @imadphp !

**Pull request merged**